ui.apps/src/main/content/jcr_root/content/themecleanflex/templates/.content.xml

```diff
@@ -10,7 +10,7 @@
     jcr:primaryType="per:PageContent"
     sling:resourceType="themecleanflex/components/page"
     jcr:title="themecleanflex root template"
-    siteCSS="[/content/themecleanflex/pages/css/palettes/default.css,/content/themecleanflex/pages/css/palette.css,/content/themecleanflex/pages/css/commons.css,/etc/felibs/themecleanflex.css,/content/themecleanflex/pages/css/custom.css,https://fonts.googleapis.com/css?family=Montserrat:300%2C300i%2C400%2C400i%2C500%2C500i%2C600%2C600i%2C700%2C700i%2C900%2C900i&amp;amp;display=swap]"
+    siteCSS="[/content/themecleanflex/pages/css/palettes/default.css,/content/themecleanflex/pages/css/palette.css,/content/themecleanflex/pages/css/commons.css,/etc/felibs/themecleanflex.css,/content/themecleanflex/pages/css/custom.css,https://fonts.googleapis.com/css2?family=Inter%3Awght%40300%3B400%3B500%3B600%3B700%3B900%26display%3Dswap]"
```

Comment:
it seems the google fonts url is incorrectly encoded resulting in a 404 when the pages are requested

Reply:
All right. I really thought I figured that out. Thanks for the check. I will fix that.

Reply:
@reusr1 I updated the string, would you mind checking again? I now see a successful network request locally.